lib/Elastica/SearchableInterface.php

```diff
@@ -26,30 +26,25 @@ interface SearchableInterface
      * }
      *
      * @param string|array|\Elastica\Query $query   Array with all query data inside or a Elastica\Query object
-     * @param null                         $options
-     * @param string                       $method  OPTIONAL Request method (use const's) (default = Request::POST)
-     *
-     * @return \Elastica\ResultSet with all results inside
+     * @param int|array                    $options Limit or associative array of options (option=>value)
```

Comment:
```suggestion
     * @param int|array|null                $options Limit or associative array of options (option=>value)
```

Not sure if we document when it can be null?

Reply:
Hum, not really.
btw: I don't really like such mix of different types to be accepted by those methods... any inputs?

Reply:
Allowing only array is possible, and removing support for the int?
But I'm not confident with the impact

Reply:
on the same line: what do you think about the `string|array|AbstractQuery` on some methods? Should we be consistent and *only* allow object of AbstractQuery to be passed in?